### PR TITLE
Amazon-AWS-EC2-Tags

### PR DESCRIPTION
**Pull request by mihaitusa**
Added create tag operation.
Signed-off-by: Mihail-Cristian Tusa <mihail-cristian.tusa@hpe.com>

### CODE REVIEW

**Comment by mihaitusa** [comment]
@gituser173  Please review it!

**Comment by gituser173** [comment]
please add unit tests

**Comment by mihaitusa** [comment]
@gituser173 Added unit tests.

File: cs-jClouds/src/main/java/io/cloudslang/content/jclouds/utils/InputsUtil.java [hunk 1]
```diff
@@ -52,7 +52,7 @@ public static int getValidInstanceStateCode(String input) {
     }
 
     public static boolean getImageNoRebootFlag(String input) {
-        return StringUtils.isBlank(input) || Boolean.parseBoolean(input);
+        return !isRelevant(input) || StringUtils.isBlank(input) || Boolean.parseBoolean(input);
```
Comment (gituser173) [inline cs-jClouds/src/main/java/io/cloudslang/content/jclouds/utils/InputsUtil.java]: isBlank can be removed

File: cs-jClouds/src/main/java/io/cloudslang/content/jclouds/utils/InputsUtil.java [hunk 2]
```diff
@@ -130,4 +130,8 @@ private static String getValidationException(String input, boolean invalid) {
         }
         return "Incorrect provided value: " + input + " input. The value doesn't meet conditions for general purpose usage.";
     }
+
+    private static boolean isRelevant(String input){
```
Comment (gituser173) [inline cs-jClouds/src/main/java/io/cloudslang/content/jclouds/utils/InputsUtil.java]: rename to isTrueOrFalse

**Pull request merged**